docs/documentation/aurora-config.md

```diff
@@ -148,7 +148,7 @@ and _app_ files will be describe in a section called "Application files".
 | permissions/edit                |          |              | No           | The groups in OpenShift that will have the edit role for the given project. Can either be an array or a space delimited string. This option must be specified in either global or env file.                                                                                    |
 | permissions/adminServiceAccount |          |              | No           | The service accounts in OpenShift that will have the admin role for the given project. Can either be an array or a space delimited string. This option must be specified in either global or env file.                                                                         |
 | globalFile                      | No       | about.yaml   | globalFile   | Replaces the global file of the project. Note that the default file is the _global_ about file. This option can only be specified in either the _base_ file or _env_ file.                                                                                                     |
-| dataclassification              | No       |              | No           | Sets data classification on the namespace, based on the sensitivity level of the data present in the environment. The value can be set to either synt, skarp, or anon and cannot be changed once set. If it needs to be changed, assistance from an administrator is required. |
+| dataclassification              | Yes *     |              | No           | Sets data classification on the namespace, based on the sensitivity level of the data present in the environment. The value can be set to either synt, skarp, or anon and cannot be changed once set. If it needs to be changed, assistance from an administrator is required. * Will be required from migration to Bare Metal clusters|
```

Comment:
Er det bedre med Will be required for migration to Bare Metal clusters?

Reply:
Det er jo ikke bare migreringen men også for nye miljøer og nye clustre etter dette. Det blir jo ny standard **fra** migreringen til Bare metal, ikke bare **for** migreringen.